project/Dependencies.scala

```diff
@@ -5,9 +5,10 @@ object Dependencies {
 
   final val scalaCheck12Version = "1.12.6"
   final val scalaCheck13Version = "1.13.4"
+  final val scalaCheck14Version = "1.14.0"
```

Comment:
Does your code share something informative if someone chooses a version like "1.13.1" ("3.0.2" for the ScalaTest definitions below)?

While there might not be a "1.13.1" today, and development might be on "1.14.x", there is the possibility of the author putting out a hotfix to the "1.13.x" line (e.g. like pluging a security hole).

Reply:
I'm not sure what you mean, but I'm assuming that ScalaCheck is binary compatible between minor / patch versions (whatever the 3rd number is). If you need to upgrade, you can rely on eviction of the older version brought in by this library.

Reply:
Good point, agreed.